docker_autoscaler.tf

```diff
@@ -6,9 +6,47 @@
 resource "aws_security_group" "docker_autoscaler" {
   count = var.runner_worker.type == "docker-autoscaler" ? 1 : 0
 
-  description = "Docker autoscaler security group"
+  name_prefix = "${local.name_sg}-docker-autoscaler"
   vpc_id      = var.vpc_id
-  name        = "${local.name_sg}-docker-autoscaler"
+  description = "Docker-autoscaler security group"
+
+  dynamic "egress" {
```

Comment:
issue: can we move this to an `aws_vpc_security_group_egress_rule` resource? See {aws_security_group](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group.html)